src/api/app/views/webui2/webui/search/owner.html.haml

```diff
@@ -4,3 +4,47 @@
   = render partial: 'tabs'
   .card-body
     %h3 Search for people responsible for package:
+    .d-flex.justify-content-center
+      = form_tag(search_owner_path, method: :get, class: 'my-3 w-75') do
+        %input{ name: 'owner', type: 'hidden', value: '1' }
+        %input{ name: 'package', type: 'hidden', value: '0' }
+        %input{ name: 'project', type: 'hidden', value: '0' }
```

Comment:
why this 0 value?

Reply:
I thought it was needed due to the `set_parameters` callback in the `search_controller`, but it isn't. I will remove them, except for `owner`, it's needed. We would need to refactor the `search_controller` to get rid of this.